### PR TITLE
[Logs] Reorganization of Logs docs

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: content/cloudflare-one/analytics/logs/audit-logs.md [hunk 1]
```diff
@@ -149,4 +149,4 @@ Cloudflare Logpush pushes Enterprise customers' HTTP request logs, including Acc
 
 For instructions on setting up Logpush, refer to [Manage the Logpush API](/logs/get-started/enable-destinations/).
 
-For more on exporting per-request Access logs, refer to [Understanding the Logpush API](/logs/reference/logpush-api-configuration/).
+For more on exporting per-request Access logs, refer to [Understanding the Logpush API](/logs/get-started/api-configuration/).
```
Comment (pedrosousa) [inline content/cloudflare-one/analytics/logs/audit-logs.md]:
```suggestion
For more on exporting per-request Access logs, refer to [API configuration](/logs/get-started/api-configuration/).
```
I couldn't find the previous name in the destination page.

File: content/logs/get-started/enable-destinations/datadog/index.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ To create a job, make a `POST` request to the Logpush jobs endpoint with the fol
 ```
 
 - **dataset** - The category of logs you want to receive. Refer to [Log fields](/logs/reference/log-fields/) for the full list of supported datasets.
-- **logpull_options** (optional) - To configure fields, sample rate, and timestamp format, refer to [Logpush API options](/logs/reference/logpush-api-configuration/#options).
+- **logpull_options** (optional) - To configure fields, sample rate, and timestamp format, refer to [Logpush API options](/logs/get-started/api-configuration/#options).
```
Comment (pedrosousa) [inline content/logs/get-started/enable-destinations/datadog/index.md]: Should the link text be just "API options", now that you removed "Logpush" from the page title?
This link text appears several times in the Logs documentation.

File: content/logs/get-started/enable-destinations/http/index.md [hunk 3]
```diff
@@ -12,14 +12,14 @@ Note that when using Logpush to HTTP endpoints, Cloudflare customers are expecte
...
-To create a Logpush job, make a `POST` request to the [Logpush job creation endpoint URL](/logs/reference/logpush-api-configuration/) with the appropriate parameters.
+To create a Logpush job, make a `POST` request to the [Logpush job creation endpoint URL](/logs/get-started/api-configuration/) with the appropriate parameters.
 
 The supported parameters are as follows:
 
 - Fields that are unchanged from other sources:
     - **dataset** (required): For example, `http_requests`.
     - **name** (optional): We suggest using your domain name as the job name.
-    - **logpull_options** (optional): Refer to [Logpush API configuration options](/logs/reference/logpush-api-configuration/#options) to configure fields, sample rate, and timestamp format.
+    - **logpull_options** (optional): Refer to [Logpush API configuration options](/logs/get-started/api-configuration/#options) to configure fields, sample rate, and timestamp format.
```
Comment (pedrosousa) [inline content/logs/get-started/enable-destinations/http/index.md]: Consider updating the link text here also.

File: content/logs/get-started/enable-destinations/other-providers/index.md [hunk 4]
```diff
@@ -31,4 +31,4 @@ To get started:
     - Zone ID
     - Destination access details for your cloud service provider
 
-2.  Configure your Logpush job. For more information on how to configure a Logpush job, refer to [Logpush API configuration](/logs/reference/logpush-api-configuration/).
+2.  Configure your Logpush job. For more information on how to configure a Logpush job, refer to [Logpush API configuration](/logs/get-started/api-configuration/).
```
Comment (pedrosousa) [inline content/logs/get-started/enable-destinations/other-providers/index.md]:
```suggestion
2.  Configure your Logpush job. For more information on how to configure a Logpush job, refer to [API configuration](/logs/get-started/api-configuration/).
```

**Review by pedrosousa — APPROVED** [review]

File: content/logs/get-started/enable-destinations/datadog/index.md (hunk 2 above)
Comment (angelampcosta) [reply content/logs/get-started/enable-destinations/datadog/index.md]: Thanks! I'll make the change.

**Review by tanushree-sharma — APPROVED** [review]

**Comment by tanushree-sharma** [comment]
Looks amazing, thank you Angela!!

**Pull request merged**